### ISSUE TITLE
Loss Logging on Visdom/tensorboard

### ISSUE THREAD

**Issue by DecentMakeover** [yhenon/pytorch-retinanet#9]
Hi Again , 
Is there a provision to visualize the loss using tensorboard or visdom or do i have to include that in the script?

Thanks

**Comment by DecentMakeover** [comment]
Successfully added tensorboard logger to the train file,Thanks

**Comment by yhenon** [comment]
This would make a nice PR if you're interested.

**Comment by DecentMakeover** [comment]
I am interested , but im not sure what im supposed to do?

**Comment by vinhbt0301** [comment]
> Successfully added tensorboard logger to the train file,Thanks

Can I have your added logger? Thank you so much

**Comment by AdityaKane2001** [comment]
Hi,
I am new to this repository and TensorBoard in general. Can you please tell how I,too can enable Tensorboard ? I am training on a custom dataset in colab
